app/services/import_polygons.rb

```diff
@@ -0,0 +1,40 @@
+class ImportPolygons
+  attr_reader :location_type
+
+  def initialize(location_type:)
+    @location_type = location_type
+  end
+
+  def call
+    response = HTTParty.get(LOCATION_POLYGON_SETTINGS[location_type][:api])
+
+    response.fetch('features', []).each do |region_response|
+      region_name = region_response.dig('attributes', LOCATION_POLYGON_SETTINGS[location_type][:name_key]).downcase
+
+      next unless location_categories_include?(region_name)
+
+      geometry_rings = region_response.dig('geometry', 'rings')
+
+      # The first ring tends to contain far more points than subsequent rings.
+      # Boundary should be visualised to check how it should be used.
+      # If algolia searches by polygon are slow, these boundaries could be downsampled significantly.
+      points = []
+      geometry_rings[0].each do |point|
+        # API returns coords in an unconventional lng,lat order
+        points.push(*point.reverse.map { |coord| coord.round(13) })
```

Comment:
Might be worth adding a comment about why we're rounding the float and why 13 decimal places rather than any other number